core/ui/src/commonMain/kotlin/org/mifospay/core/ui/utils/BaseViewModel.kt

```diff
@@ -41,7 +41,7 @@ abstract class BaseViewModel<S, E, A>(
     /**
      * A helper that returns the current state of the view model.
      */
-    protected val state: S get() = mutableStateFlow.value
+    val state: S get() = mutableStateFlow.value
```

Comment:
don't change the visibility modifier. we want only the class extend this `BaseViewModel' have only access to it